tidb-cloud/manage-user-access.md

```diff
@@ -45,6 +45,16 @@ To change the password, take the following steps:
 
 If your password is not changed within 90 days, you will get a prompt to reset your password when you log in to TiDB Cloud. It is recommended that you follow the prompt to reset the password.
 
+## Manage user profiles
+
+In TiDB Cloud, you can easily mange your profile, including your first name, last time, company name, country, and phone number.
+
+1. Click the account name in the upper-right corner of the window.
```

Comment:
Users might not read this section from Line 1 to Line 52. I am wondering whether it is necessary to clarify what "the window" is...

Reply:
"the window" means the window of the TiDB Cloud console, this is the only window used in TiDB Cloud docs.

Reply:
"the upper-right corner of the TiDB Cloud console" or "the upper-right corner of the console" LGTM